src/transformers/configuration_utils.py

```diff
@@ -190,6 +190,8 @@ class PretrainedConfig(PushToHubMixin):
     """
 
     model_type: str = ""
+    base_config_key: str = ""
+    sub_configs: List[str] = []
```

Comment:
we need to document these as well!
```suggestion
    base_config_key: str = ""
    sub_configs: Dict[str, PreTrainedConfig] = []
```